### PR TITLE
Core/Arena: more arena configs

### PR DESCRIPTION
**Pull request by FrancescoBorzi**
This PR allows the user to configure the arena rating modifiers and the arena point rate.

### CODE REVIEW

**Comment by lyrl** [comment]
good On 2015-12-14 09:00 , Francesco Borzì Wrote: This PR allows the user to configure the arena rating modifiers. You can view, comment on, or merge this pull request online at: https://github.com/TrinityCore/TrinityCore/pull/15995 Commit Summary Core/Arena: made rating modifiers configurable File Changes M src/server/game/Battlegrounds/ArenaTeam.cpp (20) M src/server/game/World/World.cpp (4) M src/server/game/World/World.h (4) M src/server/worldserver/worldserver.conf.dist (33) Patch Links: https://github.com/TrinityCore/TrinityCore/pull/15995.patch https://github.com/TrinityCore/TrinityCore/pull/15995.diff — Reply to this email directly or view it on GitHub.

File: src/server/game/Battlegrounds/ArenaTeam.cpp [hunk 1]
```diff
@@ -683,21 +685,27 @@ int32 ArenaTeam::GetRatingMod(uint32 ownRating, uint32 opponentRating, bool won
...
-        if (ownRating < 1000)
-            mod = 48.0f * (won_mod - chance);
+        if (ownRating < 1300)
+        {
+            float win_rating_modifier1 = sWorld->getFloatConfig(CONFIG_ARENA_WIN_RATING_MODIFIER_1);
+
+            if (ownRating < 1000)
+                mod =  win_rating_modifier1 * (1.0f - chance);
+            else
+                mod = ((win_rating_modifier1 / 2) + ((win_rating_modifier1 / 2) * (1300.0f - float(ownRating)) / 300.0f)) * (1.0f - chance);
```
Comment (jackpoz) [inline src/server/game/Battlegrounds/ArenaTeam.cpp]: there are a couple of int "2" here

File: src/server/game/World/World.cpp [hunk 2]
```diff
@@ -1046,6 +1047,10 @@ void World::LoadConfigSettings(bool reload)
     m_int_configs[CONFIG_ARENA_START_MATCHMAKER_RATING]              = sConfigMgr->GetIntDefault ("Arena.ArenaStartMatchmakerRating", 1500);
     m_bool_configs[CONFIG_ARENA_SEASON_IN_PROGRESS]                  = sConfigMgr->GetBoolDefault("Arena.ArenaSeason.InProgress", true);
     m_bool_configs[CONFIG_ARENA_LOG_EXTENDED_INFO]                   = sConfigMgr->GetBoolDefault("ArenaLog.ExtendedInfo", false);
+    m_float_configs[CONFIG_ARENA_WIN_RATING_MODIFIER_1]              = sConfigMgr->GetFloatDefault ("Arena.ArenaWinRatingModifier1", 48.0f);
```
Comment (Shauren) [inline src/server/game/World/World.cpp]: dat space after GetFloatDefault

**Comment by tkrokli** [comment]
:+1: Nice work, ShinDarth! Just sorry that I won't be able to test it much, though.

**Pull request merged**